src/App.tsx

```diff
@@ -1,10 +1,66 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import cn from 'classnames';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 10995;
+
+enum FilterOptions {
+  ALL = 'All',
+  ACTIVE = 'Active',
+  COMPLETED = 'Completed',
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [filterMethod, setFilterMethod] = useState<string>(FilterOptions.ALL);
+  const [isHidden, setIsHidden] = useState(false);
+
+  let visibleTodos = [...todos];
+
+  switch (filterMethod) {
+    case FilterOptions.COMPLETED:
+      visibleTodos = visibleTodos.filter(todo => todo.completed === true);
+      break;
+
+    case FilterOptions.ACTIVE:
+      visibleTodos = visibleTodos.filter(todo => todo.completed !== true);
+      break;
+
+    default:
+      break;
+  }
```

Comment:
It would be nice to use `useMemo` hook (which will depend on the filter option) and move `switch` into callback function

https://react.dev/reference/react/useMemo#usememo